iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/AbstractIdentifiableAdderAdapter.java

```diff
@@ -67,7 +69,7 @@ protected void checkAndSetUniqueId() {
             // Check Id is unique in all merging view
             if (getIndex().contains(id)) {
                 throw new PowsyblException("The network already contains an object '"
-                                           + getDelegate().getClass().getSimpleName()
+                                           + getIndex().get(n -> n.getIdentifiable(id), UnaryOperator.identity()).getClass().getSimpleName()
```

Comment:
I think it's not working well if the identifiable has been wrap into a MergedLine

Reply:
Use `getIndex().getView().getIdentifiable()` instead